src/app/tests/TestConcreteAttributePath.cpp

```diff
@@ -77,6 +115,14 @@ const nlTest sTests[] = {
                 TestConcreteDataAttributePathMatchesConcreteAttributePathEquality),
     NL_TEST_DEF("TestConcreteDataAttributePathMatchesConcreteAttributePathInequality",
                 TestConcreteDataAttributePathMatchesConcreteAttributePathInequality),
+    NL_TEST_DEF("TestConcreteDataAttributePathEqualityDefaultConstructor", TestConcreteDataAttributePathEqualityDefaultConstructor),
```

Comment:
could we get some inequality tests too? Right now this seems to pass if we implement ` bool operator==(...) { return true; }`

Reply:
Done.